code/go/internal/validator/folder_item_content.go

```diff
@@ -49,6 +49,7 @@ func loadItemContent(itemPath, mediaType string) ([]byte, error) {
 			return nil, errors.Wrapf(err, "converting YAML file to JSON failed (path: %s)", itemPath)
 		}
 	case "application/json": // no need to convert the item content
+	case "text/markdown":
```

Comment:
Add the same comment here as you did for `application/json`?

Reply:
I added a comment, but not the exact (the reason is a bit different).